shared/src/commonMain/kotlin/org/hyperskill/app/placeholder_new_user/presentation/PlaceholderNewUserActionDispatcher.kt

```diff
@@ -35,7 +35,7 @@ class PlaceholderNewUserActionDispatcher(
                     .filter { it.isBeta.not() }
 
                 val trackProgressById = progressesInteractor
-                    .getTracksProgresses(tracks.map { it.id })
+                    .getTracksProgresses(tracks.map { it.id }, forceLoadFromRemote = action.forceUpdate)
```

Comment:
Maybe it isn't necessary to add force update logic here. Initialize message is send only when screen is opened first time and after network error reloading. In both cases we should load from remote, so we can use `true` here, without forceUpdate param